### PR TITLE
Add anchor links to home page

### PR DESCRIPTION
**Pull request by AlleyWei**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by AlleyWei** [comment]
Not sure if my proposed changes are done correctly, but please let me know if there as another way it could have been written. Thank you!

File: microsite/pages/en/index.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ class Index extends React.Component {
                 src={`${baseUrl}animations/backstage-service-catalog-icon-1.gif`}
               />
               <Block.Subtitle>
-                Backstage Service Catalog{' '}
+                [Backstage Service Catalog](#backstage-service-catalog)
```
Comment (stefanalund) [inline microsite/pages/en/index.js:98]: These needs to be regular HTML links (and not Markdown formated). Example:

```
<a href="#backstage-service-catalog">the link title</a>
```

**Review by stefanalund — CHANGES_REQUESTED** [review]

File: microsite/pages/en/index.js (hunk 1 above)
Comment (AlleyWei) [reply microsite/pages/en/index.js]: Okay, gave it another shot. Did that work?

**Review by stefanalund — CHANGES_REQUESTED** [review]
Almost there. Just remove the [text] and only keep the <a...> tag.

**Comment by AlleyWei** [comment]
Got it, because the title is already in that <a...> tag?

**Comment by stefanalund** [comment]
Sorry but this does not seem to work 😞 Also, the styling will need more work:

Old:
Screenshot 2020-09-08 at 16 50 37

New:
Screenshot 2020-09-08 at 16 50 43

@AlleyWei I think we may close this PR?

**Pull request closed without merging**